content/en/documentation/for-adopters/_index.md

```diff
@@ -47,7 +47,7 @@ The Data Plane is responsible for executing data transfers, which are managed by
 data using specialized technology such as a messaging system or data integration platform. EDC includes the *Data Plane
 Framework (DPF)* for building custom Data Planes. Alternatively, a Data Plane can be built using other languages or
 technologies and integrated with the EDC Control Plane by implementing
-the [Data Plane Signaling API](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/data-plane-signaling/data-plane-signaling.md).
+the [Data Plane Signaling API](../for-contributors/data-plane/data-plane-signaling/_index.md).
```

Comment:
```suggestion
the [Data Plane Signaling API](/documentation/for-contributors/data-plane/data-plane-signaling/).
```

As Docsy suggest to "[Specify these links relative to the site’s root URL](https://www.docsy.dev/docs/best-practices/site-guidance/#linking)"

I know that is not like this everywhere, but we can slowly adapt to the convention